lib/api/policy.go

```diff
@@ -93,6 +93,10 @@ type PolicySpec struct {
 	// 	deployment != "dev"
 	// 	! has(label_name)
 	Selector string `json:"selector" validate:"selector"`
+
+	// Indicates, if True, that the rules in this policy should be applied before any data plane
+	// connection tracking, and that packets allowed by these rules should not be tracked.
+	Untracked bool `json:"untracked,omitempty" validate:"omitempty"`
```

Comment:
Minor point, but I think as far as naming goes we wan't to keep fields declarative.

Untracked feels more like a status rather than a declaration of intent.  My preference would be for something like 

```go
DontTrack bool
```

or 

```go
DoNotTrack bool
```

or similar